plugin/tsconfig.json

```diff
@@ -1,8 +1,10 @@
 {
+  "extends": "../tsconfig.json",
   "compilerOptions": {
-    "baseUrl": ".",
+    "baseUrl": "./src",
     "paths": {
-      "~/*": ["src/*"]
+      "~/*": ["*"]
```

Comment:
Thank goodness haha! Does this mean we can command click on imports again?? :D

Reply:
yes!! seems that way so far. and it also works with automatic local module imports too 💯